mirascope/core/base/message_param.py

```diff
@@ -74,3 +74,29 @@ class BaseMessageParam(BaseModel):
 
     role: str
     content: str | list[TextPart | ImagePart | AudioPart | CacheControlPart]
+
+
+Messages: TypeAlias = list[BaseMessageParam]
+
+
+class Message:
```

Comment:
`SystemMessage` would be more appropriate, but is the idea of `Messages.System` from accessibility?